tests/test-class-opengraph.php

```diff
@@ -285,7 +291,7 @@ public function test_image_IS_SINGULAR_and_HAS_open_graph_image() {
 	public function test_image_IS_SINGULAR_and_HAS_open_graph_image_AND_HAS_content_images() {
 		$post_id = $this->factory->post->create(
 			array(
-				'post_content' =>  '<img class="alignnone size-medium wp-image-490" src="' . get_site_url() . '/wp-content/plugins/wordpress-seo/tests/yoast.png" />'
+				'post_content' => '<img class="alignnone size-medium wp-image-490" src="' . get_site_url() . '/wp-content/plugins/wordpress-seo/tests/yoast.png" />'
```

Comment:
Doesn't this still fail when your wp-content is elsewhere? Shouldn't we use `content_url()` ?